CMakeLists.txt

```diff
@@ -152,7 +167,10 @@ add_executable (swig
 if (PCRE2_FOUND)
   target_link_libraries (swig ${PCRE2_LIBRARIES})
 endif ()
-install (TARGETS swig DESTINATION bin)
+if (SWIG_LIB_RELATIVE_TO_EXE)
+  target_link_libraries(swig ${CMAKE_DL_LIBS})
+endif()
+install (TARGETS swig DESTINATION bin) # TODO: seems like the executable in the zipped swigwin is actually installed at the root...
```

Comment:
* [ ] TODO: what's the actual behavior that's wanted here? Install swig(.exe) in bin/ on all platforms? Or root on windows?